prod.config.json

```diff
@@ -21,14 +24,20 @@
     "siteId": 13
   },
   "language": {
-    "siteTitle": "SIA - Signalen Informatievoorziening Amsterdam",
-    "shortTitle": "SIA",
+    "avgDisclaimer": "De gemeente Amsterdam verwerkt uw persoonsgegevens op een zorgvuldige en veilige manier in overeenstemming met de geldende wet- en regelgeving, waaronder de Algemene Verordening Gegevensbescherming. Lees meer hierover op <a href='##AVG_DISCLAIMER_URL##'>Privacyverklaring Meldingen openbare ruimte en overlast</a>.",
+    "consentToContactSharing": "Ja, ik geef de gemeente Amsterdam toestemming om mijn contactgegevens te delen met andere organisaties, als dat nodig is om mijn melding goed op te lossen.",
     "footer1": "Bel het Gemeentelijk informatienummer: <a href='tel:##GENERAL_PHONE_NUMBER_FORMATTED##'>##GENERAL_PHONE_NUMBER##</a>",
     "footer2": "op werkdagen van 08.00 tot 18.00 uur.",
```

Comment:
can we change this name to a more descriptive one?

Reply:
Will do. Next release.